doc/helpful_discussions.rst

```diff
@@ -0,0 +1,50 @@
+===================
+Helpful Discussions
+===================
+
+Pure CF outperforming Hybrid
```

Comment:
:+1: 

It may also be useful to structure this section as a FAQ. For example: "Adding user/item features makes my model perform worse than without features." + link to the issues.